net/freeradius/src/opnsense/service/templates/OPNsense/Freeradius/radiusd.conf

```diff
@@ -0,0 +1,92 @@
+{% if helpers.exists('OPNsense.freeradius.general.enabled') and OPNsense.freeradius.general.enabled == '1' %}
+
+prefix = /usr/local
+exec_prefix = ${prefix}
+sysconfdir = ${prefix}/etc
+localstatedir = /var
+sbindir = ${exec_prefix}/sbin
+logdir = /var/log
+raddbdir = ${sysconfdir}/raddb
+radacctdir = ${logdir}/radacct
+name = radiusd
+confdir = ${raddbdir}
+modconfdir = ${confdir}/mods-config
+certdir = ${confdir}/certs
+cadir   = ${confdir}/certs
+run_dir = ${localstatedir}/run/${name}
+db_dir = ${raddbdir}
+libdir = /usr/local/lib/freeradius-3*
```

Comment:
Can we have a comment here that explains our trouble with the libdir and why we use the wildcard?

Reply:
Good idea, added a comment